#!/usr/bin/python3

def raise_exception_msg(message=""):
    # This function raises a NameError exception with the given message.
    raise NameError(message)
